content/applications/general/iot/config/troubleshooting.rst

```diff
@@ -343,6 +343,66 @@ Save`.
              :align: center
              :alt: Receipt format using naming convention: EpsonTMU220B__IMC_LDV_LDH_SCALE35__.
 
+DYMO LabelWriter 450 DUO print delay
```

Comment:
Note that I am not sure if it impacts other DYMO printers, might be better to be more generic and tell that the 450 DUO is the one that we support and on which it was tested on